PennMobile/Dining/Views/DiningBalanceCell.swift

```diff
@@ -12,6 +12,7 @@ class DiningBalanceCell: UITableViewCell {
     
     static let identifier = "diningBalanceCell"
     static let cellHeight: CGFloat = 120
+    var viewController: DiningViewController?
```

Comment:
Pretty sure this is going to create a strong reference cycle, @joshdoman can you confirm? We should use `weak` I think

Reply:
Yes, this will create a retain cycle and ```weak``` should be used. Moreover, you should never pass a full view controller to a cell. This is what delegate's are for.